src/ImportPage.js

```diff
@@ -81,7 +70,8 @@ export default function ImportPage({onDataImported}: Props) {
                   </a>{' '}
                   from Chrome Devtools.
                   <br />
-                  To zoom, scroll while holding down <kbd>Ctrl</kbd> or <kbd>Shift</kbd>
+                  To zoom, scroll while holding down <kbd>Ctrl</kbd> or{' '}
```

Comment:
This is from #89's commit, that's also in this PR.

@kartikcho I'll be trying to rebase merge this PR, to see if #89's commit will appear twice on master (let's hope it doesn't). If #89's commit appears twice, I'll force push to the remaining PRs in this stack before merging them, to prevent polluting master.

Reply:
Update: Only one commit appeared on master :D We're all good